### PR TITLE
Arduinio led

### PR DESCRIPTION
**Pull request by Math-ias**
Add Michael and Junaid's LED code.

### CODE REVIEW

File: src/org/robockets/tshirtcannon/Robot.java [hunk 1]
```diff
@@ -17,8 +21,13 @@
 
 	public static OI oi;
 
+	public static Subsystem exampleSubsystem;
```
Comment (simon-andrews) [inline src/org/robockets/tshirtcannon/Robot.java:24]: Remove this

File: src/org/robockets/tshirtcannon/commands/Cylon.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package org.robockets.tshirtcannon.commands;
+
+import org.robockets.tshirtcannon.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class Cylon extends Command {
```
Comment (simon-andrews) [inline src/org/robockets/tshirtcannon/commands/Cylon.java:10]: What does this class do?

File: src/org/robockets/tshirtcannon/commands/ExampleCommand.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+package org.robockets.tshirtcannon.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+import org.robockets.tshirtcannon.Robot;
+
+/**
+ *
+ */
+public class ExampleCommand extends Command {
```
Comment (simon-andrews) [inline src/org/robockets/tshirtcannon/commands/ExampleCommand.java:11]: Remove this.

File: src/org/robockets/tshirtcannon/subsystems/ExampleSubsystem.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
+
+package org.robockets.tshirtcannon.subsystems;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ *
+ */
+public class ExampleSubsystem extends Subsystem {
```
Comment (simon-andrews) [inline src/org/robockets/tshirtcannon/subsystems/ExampleSubsystem.java:9]: Remove this.

**Comment by simon-andrews** [comment]
Fix those issues and then it'll be good to go.

File: src/org/robockets/tshirtcannon/subsystems/LED.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package org.robockets.tshirtcannon.subsystems;
+
+import org.robockets.tshirtcannon.RobotMap;
+
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ *
```
Comment (Math-ias) [inline src/org/robockets/tshirtcannon/subsystems/LED.java:8]: Javadocs please.

**Comment by Math-ias** [comment]
It would be nice if there was a command that told the Arduino to kill the lightshow.

File: src/org/robockets/tshirtcannon/commands/Cylon.java [hunk 6]
```diff
@@ -0,0 +1,36 @@
+package org.robockets.tshirtcannon.commands;
+
+import org.robockets.tshirtcannon.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
```
Comment (Math-ias) [inline src/org/robockets/tshirtcannon/commands/Cylon.java:8]: Javadocs please.

**Pull request closed without merging**